Proxyserver.js

```diff
@@ -68,6 +70,8 @@ app.get("/oilchange", async (req, res) => {
         foursquare_results.results[1].location.formatted_address,
       rating: foursquare_results.results[1].rating / 2,
       review_count: foursquare_results.results[1].stats.total_ratings,
+      lat: foursquare_results.results[1].geocodes.main.latitude,
+      lng: foursquare_results.results[1].geocodes.main.longitude,
       recommend: "yes",
     });
     // storing nearby places with the yelp api results
```

Comment:
try implement recursion to create cleaner code as there is a ton of repetition though-out this file